src/openrct2/actions/TrackRemoveAction.cpp

```diff
@@ -43,6 +43,18 @@ uint16_t TrackRemoveAction::GetActionFlags() const
     return GameAction::GetActionFlags();
 }
 
+int32_t TrackRemoveAction::NormaliseTrackType(int32_t trackType) const
```

Comment:
Does this function really have to be a member? I would prefer it being a simple static function as nothing else will use this so its not really a good fit as a member.

Reply:
Agree on making it a static function.